packages/react-components/src/components/SendBox.tsx

```diff
@@ -131,10 +121,11 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   };
 
   const textTooLongMessage = textValueOverflow ? TEXT_EXCEEDS_LIMIT : undefined;
+  const errorMessage = systemMessage ?? textTooLongMessage;
```

Comment:
Should we rename the props systemMessage to errorMessage?

Reply:
Hmmm MessageThread also uses systemMessage though and I think we should keep it consistent. Should we also change MessageThread's systemMessage to errorMessage? I would be down to do that just want to know others' thoughts.